apps/web/src/design-system/table/Table.tsx

```diff
@@ -183,7 +183,8 @@ export function Table<T extends object>({
       {!loading && pagination && pageSize > 1 && pagination?.minimalPagination && (
         <div
           style={{
-            marginTop: '10px',
+            marginTop: 'auto',
+            marginBottom: '10px',
```

Comment:
maybe let's not tackle this problem right now, because it doesn't look well on the bigger screens, we would need to think about that with Nikolay

Reply:
Will remove this for now

Reply:
done